tome-app/src/Container.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
```

Comment:
WIth Vite you don't have to import React in every component. If you need a useEffect hook or something else from React you do need to import that. It's not really hurting anything, but just an fyi

You could just have done `import {useState} from 'react'`